code/modules/urist/mob/humanhostiles.dm

```diff
@@ -371,3 +371,56 @@
 		new corpse (src.loc)
 	qdel(src)
 	return
+
+// **  HOLONAUT MOBS ** //
+/mob/living/simple_animal/hostile/urist/hololab
+	name = "Holo Lab hostile mob"
+	desc = "You shouldn't see me, report this to an admin if you do!"
+	icon = 'icons/uristmob/simpleanimals.dmi'
+	icon_dead = "holonaut_dead"
+	faction = "holonaut"
+	speak = list("...?<>D-d-d-£$D<>Id it()<> work?", "I<> T-<>%$£I'm b-<C-cant feel <>ANYT-thing.</C-cant>", "?:PO$&<S-Someone...", "Three. Nine. <<><Seven-n-n-n.. One..$£% Five Nine.")
```

Comment:
the << in this is triggering travis. either adjust the number of << allowed or change the one here
```suggestion
	speak = list("...?<>D-d-d-£$D<>Id it()<> work?", "I<> T-<>%$£I'm b-<C-cant feel <>ANYT-thing.</C-cant>", "?:PO$&<S-Someone...", "Three. Nine. ><><Seven-n-n-n.. One..$£% Five Nine.")
```